packages/SwingSet/test/bundling/test-bundles-kernel.js

```diff
@@ -46,25 +40,26 @@ test('install bundle', async t => {
   const badVersion =
     'b2-00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000';
   await t.throwsAsync(() => kernel.installBundle(badVersion, bundle), {
-    message: /is not a bundleID/,
+    message: /invalid bundleID/,
   });
 
   const tooShort = 'b1-000';
   await t.throwsAsync(() => kernel.installBundle(tooShort, bundle), {
-    message: /is not a bundleID/,
+    message: /does not match bundle/,
   });
 
   // actual installation should succeed
   await kernel.installBundle(bundleID, bundle);
-  t.deepEqual(JSON.parse(kvStore.get(`bundle.${bundleID}`)), bundle);
+  t.deepEqual(bundleStore.getBundle(bundleID), bundle);
 
   // installing the same bundle twice is a NOP
   await kernel.installBundle(bundleID, bundle);
-  t.deepEqual(JSON.parse(kvStore.get(`bundle.${bundleID}`)), bundle);
+  t.deepEqual(bundleStore.getBundle(bundleID), bundle);
 
   // the kernel's installBundle() doesn't validate
   const wrong =
     'b1-00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000';
-  await kernel.installBundle(wrong, bundle);
-  t.deepEqual(JSON.parse(kvStore.get(`bundle.${wrong}`)), bundle);
+  await t.throwsAsync(() => kernel.installBundle(wrong, bundle), {
```

Comment:
Huh, this is at odds with the "the kernel's installBundle() doesn't validate" comment above. I'm happy to have `kernel.installBundle()` do validation, but we should make sure we understand where the validation is taking place. Then let's change the comment.

Reply:
Yeah, this test fragment is now redundant with the "too short" check a few lines above it.  My inclination is to just take it out.